components/data-display/member-card.tsx

```diff
@@ -50,27 +50,33 @@ export const MemberCard: FC<MemberCard> = ({
   handleRoleChange,
   handleRemoveMember,
 }) => {
-  const { isOpen, onOpen, onClose } = useDisclosure()
+  const roleDialogDisclosure = useDisclosure();
+  const removeDialogDisclosure = useDisclosure();
```

Comment:
分割代入で中身の値を取り出して使いましょう。

```suggestion
  const { isOpen: isRoleOpen, onOpen: onRoleOpen, onClose: onRoleClose } = useDisclosure()
  const { isOpen: isRemoveOpen, onOpen: onRemoveOpen, onClose: onRemoveClose } = useDisclosure()
```